test-d/foundry/common/data/data.mjs/actorGetRollData.test-d.ts

```diff
@@ -0,0 +1,21 @@
+import { expectError, expectType } from 'tsd';
```

Comment:
I'm not sure if the minimal size of this test file means it should be merged with the other existing files or if this should be expanded etc.

Reply:
I don't really think this belongs to `data.mjs`. I would probably just add the test case to the tests for the `Actor` class.